anvil/geometry.py

```diff
@@ -5,9 +5,7 @@
 
 Module containing transformations related to geometry.
 """
-from math import ceil
-from itertools import product
-
+import itertools as it
```

Comment:
I don't like aliasing modules, especially here where we use it once. If you really want to import the whole module then just keep it as itertools

```suggestion
import itertools as it
```
```suggestion
import itertools
```